client/components/spinner-button/index.jsx

```diff
@@ -51,7 +51,8 @@ export default React.createClass( {
 				{ loading &&
```

Comment:
I talked to @mtias a bit about this. The design is definitely old. I don't think we put the spinner inside the button anywhere (that I can think of). The spinner doesn't add a whole lot in either case. We could simply remove the spinner and leave the disabled button there. We could even make the disabled button's label more descriptive and say something like 'Generating Export File...'

If we got realtime progress updates on the status of the export job we could display a progress bar here—that would be the most helpful. The spinner is kind of infinite and inspecific so if we don't need to use it, let's not.

Does that help or just muddy the water?